amp_conf/htdocs/admin/libraries/BMO/Ajax.class.php

```diff
@@ -83,6 +84,12 @@ public function doRequest($module = null, $command = null) {
 		}
 
 		session_start();
+
+                // If this is a localhost request (ie this server has sent an ajax request to this server) then
+                // no need to authenticate.
+                if ($url['host'] == '127.0.0.1') $this->settings['authenticate'] = false;
```

Comment:
@xrobau @jfinstrom and I agree that we think you should still be using REMOTE_ADDR